apps/docs/src/app/core/component-docs/token/examples/tokenizer-compact-example/tokenizer-compact-example.component.html

```diff
@@ -0,0 +1,10 @@
+<form [formGroup]="tokenizerExampleForm" (ngSubmit)="onSubmit()">
+    <div fd-form-item>
+        <fd-tokenizer [compact]="true">
+            <fd-token (onCloseClick)="removeToken(token)" *ngFor="let token of tokens" [compact]="true"
+                      [readOnly]="token.readOnly">{{token.text}}</fd-token>
+            <input formControlName="inputControl" fd-form-control class="fd-input fd-tokenizer__input" [compact]="true"
```

Comment:
Can you create directive `fd-tokenizer-input`, which will allow `fd-tokenizer__input` class? We don't force users to put classes in element. Also please remove `fd-input`, because it's provided by `fd-form-control`